### PR TITLE
test unloading

### PR DESCRIPTION
**Pull request by DiKi-2707**

### CODE REVIEW

**Review by EvgeniiTebenkov — APPROVED** [review]

File: src/test/java/school/redrover/GroupCompanionsTest.java [hunk 1]
```diff
@@ -60,4 +60,23 @@ public void testKg312(){
         Assert.assertEquals("Блог", value);
     }
 
+    @Test
+    public void testDZ() throws InterruptedException {
+        WebDriver driver = getDriver();
+        driver.get("https://exp86.ru/");
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupCompanionsTest.java:66]: you can extends your class from `BaseTest ` and use `getDriver()`

File: src/test/java/school/redrover/GroupCompanionsTest.java [hunk 2]
```diff
@@ -60,4 +60,23 @@ public void testKg312(){
...
+
+        WebElement text = driver.findElement(By.name("q"));
+        text.sendKeys("Hello world");
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupCompanionsTest.java:69]: you don't need to create variables unless you use them more than once
`driver.findElement(By.name("q")).sendKeys("Hello world")`

File: src/test/java/school/redrover/GroupCompanionsTest.java [hunk 3]
```diff
@@ -60,4 +60,23 @@ public void testKg312(){
...
+
+
+        WebElement button = driver.findElement(By.name("Поиск"));
+        button.click();
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupCompanionsTest.java:73]: driver.findElement(By.name("Поиск")).click();

**Review by IanaBlinova — APPROVED** [review]

File: src/test/java/school/redrover/GroupCompanionsTest.java [hunk 4]
```diff
@@ -60,4 +60,23 @@ public void testKg312(){
...
+
+
+        WebElement linc = driver.findElement(By.className("notetext"));
+        String result = linc.getText();
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupCompanionsTest.java:77]: String result = driver.findElement(By.className("notetext")).getText();

**Review by Gr0zpu — APPROVED** [review]

**Pull request merged**